docs/pages/database-access/guides/postgres-redshift.mdx

```diff
@@ -42,9 +42,32 @@ videoBanner: UFhT52d5bYg
 
 ## Step 3/5. Create an IAM policy for Teleport
 
-Teleport needs AWS IAM permissions to be able to:
+There are two methods of IAM authentication supported by Redshift. Teleport
+requires different IAM permissions depending on which IAM authentication method
+will be used:
 
-- Manage IAM user or IAM role policies.
+<Tabs>
```

Comment:
I think it's a little difficult for me to understand how these new instructions follow the overall flow of the guide. Is the idea that, when we run the `bootstrap` command later in the guide, we need to add the permissions mentioned in one of the `TabItem`s?

Reply:
The bootstrap does both. I was trying to explain what permissions are required in each case. Alternatively, I can duplicate the rest of this step in each tab. What do you think?

Reply:
I would be more explicit that, later in the guide, we will show the user how to add the appropriate permissions. 

The current first tab, which uses the passive voice to say "appropriate ... permissions will be added" doesn't make it clear what or who is adding the appropriate permissions. 

The second tab, which says "these IAM roles must have ...", suggests that it's up to the reader to figure out how to add these permissions, and doesn't imply that we'll show how to add them in this guide.

Reply:
I decided to add a new section to explain the two different methods and how to create the IAM role for the new method. Let me know how it looks and if I should explain a little bit more in the bootstrap section.